### PR TITLE
Refresh analytics during runPostMigrations

### PR DESCRIPTION
**Pull request by rohan-bes**

### CODE REVIEW

File: packages/database/src/runPostMigration.js [hunk 1]
```diff
@@ -86,6 +86,12 @@ export const runPostMigration = async driver => {
     ),
   );
 
+  // Refresh analytics in case they've been impacted by migrations
+  const start = Date.now();
+  await driver.runSql(`SELECT mv$refreshMaterializedView('analytics', 'public', true);`);
```
Comment (rohan-bes) [inline packages/database/src/runPostMigration.js:91]: I wanted to use `AnalyticsRefresher`'s static invokation to do this, however it relies on a `TupaiaDatabase` object and the `driver` object we have here doesn't support the `executeSql` function. I didn't really wanna change things in the `AnalyticsRefresher` so we have a small amount of code duplication here instead.

**Comment by rohan-bes** [comment]
@kael89 I believe this should fix the issue you're seeing on your strive branch, but I'm testing locally in any case

**Comment by rohan-bes** [comment]
> @kael89 I believe this should fix the issue you're seeing on your strive branch, but I'm testing locally in any case

Tested this locally and it works (takes a whole minute to refresh the analytics tho, I believe this is because of just how many answers are being added)

**Review by kael89 — APPROVED** [review]
Nice fix!

**Pull request merged**